core/code/search.js

```diff
@@ -297,7 +297,7 @@ addHook('search', function(query) {
 // search for locations
 // TODO: recognize 50°31'03.8"N 7°59'05.3"E and similar formats
 addHook('search', function(query) {
-  var locations = query.term.match(/[+-]?\d+\.\d+, ?[+-]?\d+\.\d+/g);
+var locations = query.term.replaceAll(/%2[cC]/, ',').match(/[+-]?\d+\.\d+, ?[+-]?\d+\.\d+/g);
```

Comment:
Mhmm... The replacement doesn't work now 😬

Maybe try adding gi into the pattern 🤷
/%2c/gi (global+case insensitive search)


```suggestion
var locations = query.term.replaceAll(/%2c/gi, ',').match(/[+-]?\d+\.\d+, ?[+-]?\d+\.\d+/g);
```